lib/jsonnet/evaluator.rb

```diff
@@ -0,0 +1,46 @@
+module Jsonnet
+  class Evaluator
```

Comment:
Good point.

It's just because I was too lazy to call `#initialize` in C. It'd be clearer if the `vm_s_new` would call `#initialize`.
Let me try again.
Then, I'll merge this PR.